From 40a0c18779ec2723e2add2d6ab1be2dcae52fd30 Mon Sep 17 00:00:00 2001
From: John Zhuge <jzhuge@cloudera.com>
Date: Mon, 23 Jan 2017 16:31:53 -0800
Subject: [PATCH 2173/2748] Revert "Revert "HADOOP-13900. Remove snapshot
 version of SDK dependency from Azure Data Lake
 Store File System. Contributed by Vishwajeet
 Dusane""

This reverts commit 84857c904ba2df4f98fe8b8eeae3574b27a77993.

Change-Id: I1261b26ebf499b1ac2631753805310514d0464ed
---
 hadoop-tools/hadoop-azure-datalake/pom.xml |    8 --------
 1 file changed, 8 deletions(-)

diff --git a/hadoop-tools/hadoop-azure-datalake/pom.xml b/hadoop-tools/hadoop-azure-datalake/pom.xml
index b8a068b..fab4ed9 100644
--- a/hadoop-tools/hadoop-azure-datalake/pom.xml
+++ b/hadoop-tools/hadoop-azure-datalake/pom.xml
@@ -35,14 +35,6 @@
     <file.encoding>UTF-8</file.encoding>
     <downloadSources>true</downloadSources>
   </properties>
-  <repositories>
-    <repository>
-      <id>snapshots-repo</id>
-      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
-      <releases><enabled>false</enabled></releases>
-      <snapshots><enabled>true</enabled></snapshots>
-    </repository>
-  </repositories>
   <build>
     <plugins>
       <plugin>
-- 
1.7.9.5

